utils.py

```diff
@@ -117,6 +117,8 @@ def _read_builds():
     if os.environ.get("MOCK"):
         builds = "mock.builds.json"
     try:
+        if pathlib.Path(builds).stat().st_size == 0:
```

Comment:
prefer `os.path.getsize(builds)`

Reply:
Sure, although the documentation says that these return equivalent numbers:

https://docs.python.org/3/library/pathlib.html#pathlib.Path.stat
https://docs.python.org/3/library/os.html#os.stat_result.st_size
https://docs.python.org/3/library/os.path.html#os.path.getsize

I prefer `pathlib` because it is easy to reason about but I will change it.

Reply:
a5955b6c8047fe8c3769d35fc7b2126a576c1df7

Reply:
[citation needed]

Reply:
Not super authoritative but good reads:

https://towardsdatascience.com/dont-use-python-os-library-any-more-when-pathlib-can-do-141fefb6bdb5

https://stackoverflow.com/questions/67112343/pathlib-vs-os-path-join-in-python

https://treyhunner.com/2018/12/why-you-should-be-using-pathlib/

Reply:
TIL (Stupid python stdlib full of dead batteries) ok, feel free to drop a5955b6c8047fe8c3769d35fc7b2126a576c1df7 then

Reply:
@nickdesaulniers The docs say `pathlib` is the high-level interface (objects vs. strings). There is also rationale in the PEP that introduced it.